activerecord/lib/active_record/attributes.rb

```diff
@@ -74,16 +74,19 @@ module ClassMethods
       #   # db/schema.rb
       #   create_table :store_listings, force: true do |t|
       #     t.string :my_string, default: "original default"
+      #     t.string :changed_json
       #   end
       #
       #   StoreListing.new.my_string # => "original default"
       #
       #   # app/models/store_listing.rb
       #   class StoreListing < ActiveRecord::Base
       #     attribute :my_string, :string, default: "new default"
+      #     attribute :changed_json, :json, default: -> { {} }
```

Comment:
Can’t we do `default: {}` here?

Reply:
thanks comment.
It can move `defaut: {}`. update examples :)